LauncherInjector/main.cpp

```diff
@@ -336,6 +396,40 @@ int main(int argc, char* argv[])
 	}
 
 	{
+
+		wchar_t buffer[_MAX_PATH];
+		GetModuleFileNameW(NULL, buffer, _MAX_PATH); // Get full executable path
+		std::wstring w = std::wstring(buffer);
+		std::string exepath = std::string(w.begin(), w.end());			   // Convert from wstring to string
+		std::string path = exepath.substr(0, exepath.find_last_of("/\\")); // Substr to just the folder name
```

Comment:
This should check for the last `/` **before the first `.exe`**

Do not merge before this is fixed

Reply:
Upon testing, this seems to be fine